src/mon/PGMap.cc

```diff
@@ -2057,9 +2057,9 @@ void PGMap::dump_fs_stats(
         << stringify(si_t(osd_sum.kb_used*1024));
     float used = 0.0;
     if (osd_sum.kb > 0) {
-      used = ((float)osd_sum.kb_used / osd_sum.kb);
+      used = ((float)(osd_sum.kb - osd_sum.kb_avail) / osd_sum.kb);
```

Comment:
note for posterity:

the following warning mentioned in http://tracker.ceph.com/issues/19487 comes from a jewel OSD:

> 2017-04-04 20:57:40:894659 [ERR] OSD full dropping all updates 51% full

see https://github.com/ceph/ceph/commit/80829d7805b1113d322509345372bf9c7984559b#diff-fa6c2eba8356ae1442d1bf749beacfdfR621

the way how we calculate the use ratio here is different from `OSDService::check_full_status()`. that's why we have the discrepancy here.